static/js/rename.js

```diff
@@ -64,17 +59,27 @@ function rename() {
     renameError(xmlHttp);
   }
 
-  xmlHttp.send(data);
+  xmlHttp.send();
 }
 
 function renameError(xmlHttp) {
   resetRenameButton();
-  alert("error with renaming device");
-  console.log(xmlHttp);
+  alert("error with renameing device: " + getResponseMessage(xmlHttp.responseText))
 }
 
 function resetRenameButton() {
   var updateButton = document.getElementById("rename-button");
   updateButton.disabled = false;
   updateButton.innerHTML = "Rename";
 }
+
+
+function getResponseMessage(bodyString) {
+  // The body string needs to be sliced as the go-api will add a prefix onto the response from the server
```

Comment:
what's the prefix? can we get rid of it?

Reply:
This would require changing go-api. 
https://github.com/TheCacophonyProject/go-api/blob/master/api.go#L504

Reply:
ok. we can do that later